FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -339,15 +237,13 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                             <msItem xml:id="p1_i3">
                            <locus from="127vb" to="128rb"></locus>
                                 
-                            <title>An elegy on Mary taken from a homily of <persName ref="PRS5720JohnChr"></persName>.</title> 
+                                <title ref="LIT3058RepCh338" type="incomplete">An elegy on Mary taken from ʾƎsaggǝd laki ʾǝsaggǝd laki</title> 
```

Comment:
NB you may not have TITLE inside of a TITLE, the first `title `must have a `ref`